playbook/app/pb_kits/playbook/pb_form_pill/form_pill.html.erb

```diff
@@ -1,7 +1,7 @@
 <%= content_tag(:div,
     id: object.id,
     data: object.data,
-    class: object.classname) do %>
+    class: object.classname + object.size_class) do %>
```

Comment:
I could be totally wrong but I think you could get away with this in the class generator. See below...